mkosi/__init__.py

```diff
@@ -609,6 +609,9 @@ def run_configure_scripts(config: Config) -> Config:
         MKOSI_DEBUG=one_zero(ARG_DEBUG.get()),
     )
 
+    if config.architecture.to_efi() is not None:
+        env["EFI_ARCHITECTURE"] = str(config.architecture.to_efi())
```

Comment:
Just for posterity: This is a rake that's only sidestepped by the `is not None` check, since `str(None)` is equal to `"None"`.